choice.rb

```diff
@@ -0,0 +1,96 @@
+require './student'
+require './teacher'
+require './book'
+require './rental'
+
+class HandleMenuChoice
+  def initialize
+    @people = []
+    @books = []
+    @rentals = []
+  end
+
+  def list_books
+    if @books.empty?
+      puts 'There is no book registered in the library'
+    else
+      @books.each { |book| puts book }
+    end
+  end
+
+  def list_people
+    if @people.empty?
+      puts 'There is no one registered in the library'
+    else
+      @people.each { |person| puts person }
+    end
+  end
+
+  def create_person
+    print 'Do you want to create a student (1) or a teacher (2)? [Input the number]: '
+    option = gets.chomp
+    print 'Age: '
+    age = gets.chomp
+    print 'Name: '
+    name = gets.chomp
+    case option
+    when '1'
+      create_student(age, name)
+    when '2'
+      create_teacher(age, name)
+    else
+      puts 'Not a valid option'
+      return
+    end
+    puts 'Person created successfully'
+  end
+
+  def create_student(age, name)
+    print 'Has parent permission? [Y/N]: '
+    pp = gets.chomp
+    @people.push(Student.new(name: name, age: age, parent_permission: translate_answer(pp.downcase)))
+  end
```

Comment:
- [ ] CRUD operations and UI rendering are two separate features that shouldn't coexist in the same class. I'd suggest revisiting this [lesson](https://github.com/microverseinc/curriculum-ruby/blob/main/oop/lessons/solid.md#best-practices-and-solid) and making sure that you are following the `Single responsibility rule`. Kindly do the same for other CRUD operations as well.
> A class should have one and only one reason to change, meaning that a class should have only one job.